consensus/src/block_storage/sync_manager.rs

```diff
@@ -67,17 +71,21 @@ impl BlockStore {
     }
 
     /// Fetches dependencies for given sync_info.quorum_cert
-    /// If gap is large, performs state sync using process_highest_commit_cert
+    /// If gap is large, performs state sync using sync_to_highest_commit_cert
     /// Inserts sync_info.quorum_cert into block store as the last step
     pub async fn add_certs(
         &self,
         sync_info: &SyncInfo,
         mut retriever: BlockRetriever,
     ) -> anyhow::Result<()> {
-        self.sync_to_highest_commit_cert(sync_info.highest_commit_cert().clone(), &mut retriever)
-            .await?;
+        self.sync_to_highest_commit_cert(
```

Comment:
probably should rename to sync_to_highest_ordered_cert

Reply:
resolved